<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReorientImageDialog</class>
 <widget class="QDialog" name="ReorientImageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit:read-only {
  background-color:rgb(240,240,240);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0">
     <item row="7" column="1">
      <widget class="QtSimpleOpenGLBox" name="iconCurrent" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:blue;</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Voxel to World &lt;br/&gt;Matrix (NIFTI):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Voxel X Axis:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="outMatrixCurrent">
          <property name="minimumSize">
           <size>
            <width>242</width>
            <height>122</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>242</width>
            <height>122</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>60</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="outMatrixNew">
          <property name="minimumSize">
           <size>
            <width>242</width>
            <height>122</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>242</width>
            <height>122</height>
           </size>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>60</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>New Row</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Current Orientation:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>New Orientation:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QWidget" name="widget_4" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="inNewRAI">
          <property name="maxLength">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="outInvalidStatus">
          <property name="styleSheet">
           <string notr="true">color:rgb(128, 0, 0);
font: 11pt;</string>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="outCurrentAxisDirY">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QWidget" name="widget_7" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="inNewAxisDirY"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnReverseY">
          <property name="toolTip">
           <string>Reverse the direction of the axis</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources/SNAPResources.qrc">
            <normaloff>:/root/revertaxis_16.png</normaloff>:/root/revertaxis_16.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="outCurrentAxisDirZ">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QWidget" name="widget_8" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="inNewAxisDirZ"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnReverseZ">
          <property name="toolTip">
           <string>Reverse the direction of the axis</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources/SNAPResources.qrc">
            <normaloff>:/root/revertaxis_16.png</normaloff>:/root/revertaxis_16.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="outCurrentAxisDirX">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QWidget" name="widget_6" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="inNewAxisDirX"/>
        </item>
        <item>
         <widget class="QToolButton" name="btnReverseX">
          <property name="toolTip">
           <string>Reverse the direction of the axis</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources/SNAPResources.qrc">
            <normaloff>:/root/revertaxis_16.png</normaloff>:/root/revertaxis_16.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Voxel Z Axis:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QtSimpleOpenGLBox" name="iconNew" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background:blue;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Voxel Y Axis:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QWidget" name="widget_5" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLineEdit" name="outCurrentRAI">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="widget_9" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>RAI Code:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnApply">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtSimpleOpenGLBox</class>
   <extends>QWidget</extends>
   <header>QtSimpleOpenGLBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>inNewRAI</tabstop>
  <tabstop>inNewAxisDirX</tabstop>
  <tabstop>btnReverseX</tabstop>
  <tabstop>inNewAxisDirY</tabstop>
  <tabstop>btnReverseY</tabstop>
  <tabstop>inNewAxisDirZ</tabstop>
  <tabstop>btnReverseZ</tabstop>
  <tabstop>btnApply</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>outMatrixNew</tabstop>
  <tabstop>outCurrentRAI</tabstop>
  <tabstop>outCurrentAxisDirZ</tabstop>
  <tabstop>outCurrentAxisDirY</tabstop>
  <tabstop>outCurrentAxisDirX</tabstop>
  <tabstop>outMatrixCurrent</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/SNAPResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ReorientImageDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
